torch_lr_finder/lr_finder.py

```diff
@@ -348,13 +347,12 @@ def plot(self, skip_start=10, skip_end=5, log_lr=True, show_lr=None, filepath=No
             plt.axvline(x=show_lr, color="red")
 
         if filepath is not None:
-            path = pathlib.Path(filepath)
-            parent_directory = path.parents[0]
-            if not parent_directory.is_dir():
+            parent_directory = os.path.dirname(filepath)
+            if not os.path.isdir(parent_directory):
                 print(f'Creating directory "{parent_directory}" as it does not exist')
-                parent_directory.mkdir(parents=True, exist_ok=True)
-            print(f'Saving plot under "{path}"')
-            plt.savefig(path)
+                os.mkdir(parent_directory)
```

Comment:
It would be better to use `os.makedirs` instead of `os.mkdir` for multi-layer directory.
Otherwise, you have to use `try...except` to handle `FileNotFoundError` here.
(In my opinion, `os.makedirs` is better for user experience)